pysrc/fetch/dune.py

```diff
@@ -0,0 +1,112 @@
+"""
+All Dune Query executions should be routed through this file.
+# TODO - this should (perhaps) belong to dune-client package including FileIO
```

Comment:
In my opinion also here to have an issue reference would be a nice to have.

Reply:
Right. Will do - also this is outdated (since FileIO is already out of here).